src/viur/core/request.py

```diff
@@ -357,10 +359,8 @@ def _process(self):
                     if filename := conf["viur.mainApp"].render.getTemplateFileName((f"{error_info['status']}", "error"),
                                                                                    raise_exception=False):
                         template = conf["viur.mainApp"].render.getEnv().get_template(filename)
-                        res = template.render(error_info)
-
-                        # fixme: this might be the viur/core/template/error.html ...
-                        extendCsp({"style-src": ['sha256-Lwf7c88gJwuw6L6p6ILPSs/+Ui7zCk8VaIvp8wLhQ4A=']})
+                        res = template.render(error_info, nonce=(nonce := utils.generateRandomString(16)))
```

Comment:
```suggestion
                        nonce = utils.generateRandomString(16)
                        res = template.render(error_info, nonce=nonce)
```
I know this work but i think this more readable.